<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>510</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resrc/resource.qrc">
    <normaloff>:/icon/pycom</normaloff>:/icon/pycom</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>3</y>
      <width>186</width>
      <height>306</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QComboBox" name="comboBox_SPort">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>65</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>SerialPort</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>105</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>BaudRate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_BRate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>105</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_BSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>145</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>145</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ByteSize</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_SBit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>185</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>StopBits</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_PBit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>225</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>225</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ParityBits </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>CheckPort</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Check">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>25</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Open">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>265</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Close">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>265</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>315</y>
      <width>691</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Send</string>
    </property>
    <widget class="QTabWidget" name="SendTab">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>18</y>
       <width>688</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="single">
      <attribute name="title">
       <string>Single</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit_sSend">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>495</width>
         <height>106</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_sCycle">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Cycle(ms)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sCycle">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>80</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_sNewline">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>5</y>
         <width>79</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>NewLine</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_sHexmode">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>45</y>
         <width>79</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>HexMode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_sSend">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>5</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_sClear">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>45</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="multi">
      <attribute name="title">
       <string>Multi</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox_m1">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>5</y>
         <width>31</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_m1">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>5</y>
         <width>186</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_m2">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>40</y>
         <width>186</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_m2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>40</y>
         <width>31</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_m3">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>75</y>
         <width>186</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_m3">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>75</y>
         <width>31</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_m1">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>5</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_m2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_m3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>75</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_m4">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>5</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_m5">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>40</y>
         <width>31</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_m5">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_m6">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>75</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_m6">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>75</y>
         <width>31</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_m4">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>5</y>
         <width>31</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_m6">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>75</y>
         <width>186</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_m4">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>5</y>
         <width>186</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_m5">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>40</y>
         <width>186</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_mNewLine">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>0</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>NewLine</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_mHexMode">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>30</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>HexMode</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mCycle">
       <property name="geometry">
        <rect>
         <x>595</x>
         <y>80</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_mCycle">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Cycle(ms)</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="file">
      <attribute name="title">
       <string>File</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_fFile">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>20</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>36</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string> File:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_fSelect">
       <property name="geometry">
        <rect>
         <x>405</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_fSend">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>60</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>support: 1. txt file   2. customized json file</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="guide">
      <attribute name="title">
       <string>Guide</string>
      </attribute>
      <widget class="QPlainTextEdit" name="plainTextEdit_Guide">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>681</width>
         <height>103</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>3</y>
      <width>496</width>
      <height>306</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Receive</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QTextEdit" name="textEdit_Receive">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>486</width>
       <height>251</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_RHexmode">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>275</y>
       <width>79</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>HexMode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RClear">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>275</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RSave">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>275</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuEncoding_Type">
     <property name="title">
      <string>Encoding Set</string>
     </property>
     <addaction name="actionASCII"/>
     <addaction name="separator"/>
     <addaction name="actionUTF_8"/>
     <addaction name="separator"/>
     <addaction name="actionUTF_16"/>
     <addaction name="separator"/>
     <addaction name="actionUTF_32"/>
     <addaction name="separator"/>
     <addaction name="actionGBK_GB2312"/>
    </widget>
    <addaction name="menuEncoding_Type"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionUTF_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionASCII">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ASCII</string>
   </property>
  </action>
  <action name="actionUTF_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-16</string>
   </property>
  </action>
  <action name="actionUTF_32">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-32</string>
   </property>
  </action>
  <action name="actionGBK_GB2312">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GBK/GB2312</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resrc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
